### PR TITLE
Improve formatting & correct grammar

### PR DESCRIPTION
**Pull request by QwerMike**

### CODE REVIEW

File: src/System.Net.Http/src/HttpDiagnosticsGuide.md [hunk 1]
```diff
@@ -90,8 +90,8 @@ Otherwise, `Activity.Current` represent some 'parent' activity (presumably incom
 7. `DiagnosticListener.Write("System.Net.Http.HttpRequestOut.Stop", new {Response, RequestTaskStatus})` - notifies that activity (outgoing request) is stopping
 
 # Non-Activity events (deprecated)
-There are two events System.Net.Http.Request and System.Net.Http.Response, currently are also emited for compatibility purposes.  
-They are redundant with the System.Net.Http.HttpRequestOut start and stop events (but do not set Activity.Current and follow activity conventions (start/stop)
+There are two events `System.Net.Http.Request` and `System.Net.Http.Response`, currently are also emited for compatibility purposes.  
+They are redundant with the `System.Net.Http.HttpRequestOut.Start` and `System.Net.Http.HttpRequestOut.Stop` events (but do not set Activity.Current and follow activity conventions (start/stop)
```
Comment (safern) [inline src/System.Net.Http/src/HttpDiagnosticsGuide.md]: It seems like you have a missing `)` and that `Activity.Current` can be quoted as well.

**Comment by safern** [comment]
cc: @karelz

File: src/System.Net.Http/src/HttpDiagnosticsGuide.md [hunk 2]
```diff
@@ -90,8 +90,8 @@ Otherwise, `Activity.Current` represent some 'parent' activity (presumably incom
 7. `DiagnosticListener.Write("System.Net.Http.HttpRequestOut.Stop", new {Response, RequestTaskStatus})` - notifies that activity (outgoing request) is stopping
 
 # Non-Activity events (deprecated)
-There are two events System.Net.Http.Request and System.Net.Http.Response, currently are also emited for compatibility purposes.  
-They are redundant with the System.Net.Http.HttpRequestOut start and stop events (but do not set Activity.Current and follow activity conventions (start/stop)
+There are two events `System.Net.Http.Request` and `System.Net.Http.Response`, currently are also emited for compatibility purposes.
```
Comment (danmosemsft) [inline src/System.Net.Http/src/HttpDiagnosticsGuide.md]: emited -> emitted

File: src/System.Net.Http/src/HttpDiagnosticsGuide.md [hunk 3]
```diff
@@ -90,8 +90,8 @@ Otherwise, `Activity.Current` represent some 'parent' activity (presumably incom
 7. `DiagnosticListener.Write("System.Net.Http.HttpRequestOut.Stop", new {Response, RequestTaskStatus})` - notifies that activity (outgoing request) is stopping
 
 # Non-Activity events (deprecated)
-There are two events System.Net.Http.Request and System.Net.Http.Response, currently are also emited for compatibility purposes.  
-They are redundant with the System.Net.Http.HttpRequestOut start and stop events (but do not set Activity.Current and follow activity conventions (start/stop)
+There are two events `System.Net.Http.Request` and `System.Net.Http.Response`, currently are also emited for compatibility purposes.  
+They are redundant with the `System.Net.Http.HttpRequestOut.Start` and `System.Net.Http.HttpRequestOut.Stop` events (but do not set Activity.Current and follow activity conventions (start/stop)
 They are deprecated, and consumers are advised only to depend on them to suport.NET Core V1.1 apps (where the new events are not present).
```
Comment (danmosemsft) [inline src/System.Net.Http/src/HttpDiagnosticsGuide.md]: suport -> support

**Comment by danmosemsft** [comment]
Thanks @QwerMike! Would you like to try one of our up for grabs issues next?

**Comment by QwerMike** [comment]
> Thanks @QwerMike! Would you like to try one of our up for grabs issues next?

Yea, I can try (hope I can find a spare time for it).

**Comment by karelz** [comment]
cc @lmolkova @vancem

**Pull request merged**